pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -3,10 +3,17 @@
 import itertools
 
 from pulpcore.client.pulpcore import ArtifactsApi, ApiClient as CoreApiClient
-
+from pulpcore.client.pulp_rpm import (
+    DistributionsRpmApi,
+    PublicationsRpmApi,
+    RepositoriesRpmApi,
+    RpmRepositorySyncURL,
+    RemotesRpmApi,
+)
```

Comment:
If we are trying to keep imports in alphabetical order, I'd check all  imports in this file. This one is just one occurrence.